control/templates/base.html

```diff
@@ -1,115 +1,124 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
+<html lang="en" class="h-100">
```

Comment:
`h-100` is a fix for the global stylesheet change.

Reply:
Note to future readers too:

 * `h-100` combined with `mt-auto` allows the footer to create as much margin as it needs above it to make the nav sticky and nice without fixed values or heights